src/libmongoc/tests/test-mongoc-sample-commands.c

```diff
@@ -579,8 +579,8 @@ test_example_14 (mongoc_database_t *db)
    }
    /* End Example 14 */
 
+   done:
```

Comment:
```suggestion
done:
```

Suggest aligning for consistent formatting with other labels. `clang-format` normally formats, but is disabled with the `// clang-format off` comment.